analytic_partner_hr_timesheet/models/res_partner.py

```diff
@@ -1,20 +1,22 @@
 # -*- coding: utf-8 -*-
-# (c) 2015 Antiun Ingeniería S.L. - Pedro M. Baeza
+# Copyright 2015 Tecnativa - Pedro M. Baeza
+# Copyright 2017 Tecnativa - Luis M. Ontalba
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
-from openerp import models, api, fields
+
+from odoo import api, fields, models
 
 
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
-    timesheets = fields.One2many(
-        comodel_name='hr.analytic.timesheet', inverse_name='other_partner_id',
+    timesheet_ids = fields.One2many(
+        comodel_name='account.analytic.line', inverse_name='other_partner_id',
         string='Timesheet activities')
     timesheet_count = fields.Integer(
         string='Timesheet Activities Number',
         compute='compute_timesheet_count', store=True)
 
     @api.one
```

Comment:
Yes, I think so, but maybe not in this case ... from the official doc:

![api one](https://user-images.githubusercontent.com/5930419/29371348-704250e8-82a8-11e7-811b-584fa406469f.png)

Other way without @api.one could be:

    @api.depends('timesheet_ids')
    def _compute_timesheet_count(self):
        for partner in self:
            partner.timesheet_count = len(partner.timesheet_ids)




What do you think?